tests/modeling/test_hourly_model.py

```diff
@@ -8,6 +8,9 @@
 import pandas as pd
 import pytz
 import tempfile
+import eemeter.modeling.exceptions as model_exceptions
+
+MIN_CONTIGUOUS_MONTHS = 0
```

Comment:
It is generally good practice to extract constants, but I wonder if since you're using it in only one place if it wouldn't make more sense to pull it inline. That would help make it a bit easier to follow for future contributors to follow the main idea of the tests. That would also make it clear that it's not a library-wide constant that is imported, which is what it looks like if you just look at L71

Reply:
Ok,  I see. Removed it. 
nitpick: this is test module and so library wide constant is not a problem.